### ISSUE TITLE
Opening a new tab with ctrl+shift+1 provides no spoken feedback with NVDA

### ISSUE THREAD

**Issue by Neurrone** [microsoft/terminal#5936]
# Environment

```none
Windows build number: 10.0.19041.264
Windows Terminal version (if applicable): latest 0.11 preview
NVDA: latest master snapshot alpha-20220,1c4519a6
```

# Steps to reproduce

1. With NVDA running, open terminal. Press ctrl+shift+1 to open a new tab.

# Expected behavior

NVDA informs me about the newly opened tab. Output from subsequent commands in this new tab are automatically read out by NVDA.

# Actual behavior

Silence. No feedback is provided. Additionally, new text is not read until tabbing away and back into terminal (i.e, it needs to regain focus).

**Comment by codeofdusk** [comment]
Cc @carlos-zamora @DHowett-MSFT.

**Comment by DHowett** [comment]
Thanks! Triaged into near-term.

**Comment by carlos-zamora** [comment]
Looks like this got fixed in #10971 and I'm unable to get a repro of this on 1.13.10395.0 too.

I'm going to close this issue out as "Resolution-Fix-Available", but if you're still experiencing it, let me know and I'll reopen it.

**Comment by Neurrone** [comment]
I can confirm this is fixed now, thanks!
